src/conversion/format.ts

```diff
@@ -0,0 +1,78 @@
+import type { GeoJSON } from 'geojson';
+
+import { Coordinate } from '../types';
+
+// tslint:disable-next-line: no-any
+const prettyPrint = (value: any) => JSON.stringify(value, null, 2);
+
+export enum FormatType {
+    GEOJSON = 'geojson',
+    LATLNG = 'latlng',
+    JTS = 'jts'
+}
+
+export interface Format {
+    name: FormatType;
+    displayName: string;
+    description?: string;
+    serialize: (coordinates: Coordinate[]) => string;
+    deserialize: (raw: string) => Coordinate[];
+    validate: (value: string) => boolean;
+}
+
+export const format: Record<FormatType, Format> = {
```

Comment:
Having the serializer and the deserializer in a one format reference is a smart idea 👍.